File: src/main/scala/yuma140902/yumalib/api/util/StringUtils.scala
package yuma140902.yumalib.api.util

object StringUtils {
  def suffix(delimiter: String, value: String): String =
    if (value.isEmpty) ""
    else delimiter + value

  def prefix(value: String, delimiter: String): String =
    if (value.isEmpty) ""
    else value + delimiter
}

File: src/main/scala/yuma140902/yumalib/api/util/StringUtils.java
package yuma140902.yumalib.api.util;

public class StringUtils {
	public static String suffix(String sep, String value) {
		return value.isEmpty() ? "" : sep + value;
	}
	
	public static String prefix(String value, String sep) {
		return value.isEmpty() ? "" : value + sep;
	}
}
